cmd/describe_component.go

```
@@ -1,34 +1,71 @@
 package cmd
 
 import (
+	"errors"
+
 	"github.com/spf13/cobra"
 
 	e "github.com/cloudposse/atmos/internal/exec"
 	u "github.com/cloudposse/atmos/pkg/utils"
 )
 
+var errInvalidFlag = errors.New("invalid arguments. The command requires one argument `component`")
```

golangci-lint (1.64.8)

[error] var `errInvalidFlag` is unused

(unused)